build/common.sh

```diff
@@ -395,6 +395,23 @@ function kube::build::build_image() {
 # $3 is the value to set the --pull flag for docker build; true by default
 # $4 is the set of --build-args for docker.
 function kube::build::docker_build() {
+  "${DOCKER[@]}" buildx >/dev/null 2>&1
+  local exitCodeBuildx=$?
+  if [[ ${exitCodeBuildx} -ne 0 ]]; then
```

Comment:
Is there a reason to store `$?` to `exitCodeBuildx` instead of using it directly in the if-statement on line 400?